### PR TITLE
Remove all scales from CARTO fetchMap 

### PR DESCRIPTION
**Pull request by padawannn**
#### Background
Cloud-native is going to start styling using pixels instead of meters so the scales are no longer necessary
#### Change List
- Remove lineWidthScale and pointRadiusScale from CARTO fetchMap

### CODE REVIEW

File: modules/carto/src/api/layer-map.ts [hunk 1]
```diff
@@ -110,7 +107,7 @@ export function getLayer(
     },
     geojson: {
       Layer: GeoJsonLayer,
-      defaultProps: {lineWidthScale: 2}
+      defaultProps: {}
```
Comment (felixpalmer) [inline modules/carto/src/api/layer-map.ts:110]: As `defaultProps` is empty you can simply remove it

**Review by felixpalmer — CHANGES_REQUESTED** [review]

**Comment by felixpalmer** [comment]
A number of tests are also failing

**Comment by felixpalmer** [comment]
Comparing using the `test/apps/carto-map` app with map id `ba2ef0ba-e7bb-4a9a-a2a0-e8ade556b3d2` shows a difference. Is this because the Builder update is yet to be deployed?
Screenshot 2022-03-31 at 9 50 10

**Comment by padawannn** [comment]
> Comparing using the `test/apps/carto-map` app with map id `ba2ef0ba-e7bb-4a9a-a2a0-e8ade556b3d2` shows a difference. Is this because the Builder update is yet to be deployed? Screenshot 2022-03-31 at 9 50 10

Yes, exactly. The Builder PRs have not been approved yet. I'll deploy in a dedicated to check the differences.

File: modules/carto/src/api/layer-map.ts (hunk 1 above)
Comment (padawannn) [reply modules/carto/src/api/layer-map.ts]: I was not sure if I could remove it. Done

**Review by felixpalmer — APPROVED** [review]
Looks good to me. However I would say hold off on merging until we've done full QA and agreed on a deployment plan

**Comment by felixpalmer** [comment]
@padawannn this PR: https://github.com/visgl/deck.gl/pull/6800 fixes the issues found in QA. I would recommend we review and merge that first and then update this one and do a final round of QA

**Pull request merged**